### ISSUE TITLE
Challenge 5, example is a bit misleading

### ISSUE THREAD

**Issue by Roger-Heathcote** [oliverjam/learn-form-validation#5]
Challenge 5 instructs the user to create a div to use for validation error messages. In the example directly above it shows a div with the error message hard coded into the HTML...

    <div id="emailError">Please enter an email address</div>

This confused a few people as the div they need to create ought to be empty. Either the example div could be made to be empty or the instructions could explicitly say "create an empty div".

**Comment by oliverjam** [comment]
Yeah I can see how that would confuse people. The example is just there to show how the ID and aria-describedby match up, so we could make the error div empty (or contain something obvious like `{your message should go in here}`
